@navikt/ds-tokens/src/index.js

```diff
@@ -156,6 +156,13 @@ module.exports = {
       focus: { value: "{navds.color.blue.80.value}" },
       shadow: { value: "0 0 0 2px {navds.color.blue.80.value}" },
     },
+    contentContainer: {
```

Comment:
Åpent spørsmål (her har ikke vi bestemt oss enda): Når hva skal til for at verdier trekkes opp fra komponenter og defineres som tokens?

Hvordan har dere tenkt her? Er tanken at team kan ha nytte av å bruke disse tokenene direkte?

Reply:
Godt spørsmål, jeg har inge spesielle meninger her, så bare si ifra om dere ønsker å ha det som token eller legge verdiene rett i klassen :)